packages/snaps-utils/src/handlers/types.ts

```diff
@@ -42,6 +42,7 @@ export enum HandlerType {
   OnUserInput = 'onUserInput',
   OnAssetsLookup = 'onAssetsLookup',
   OnAssetsConversion = 'onAssetsConversion',
+  OnAssetHistoricalPrice = 'onAssetHistoricalPrice',
```

Comment:
Didn't we specify as `onAssetsHistoricalPrice` in the SIP?

Reply:
Nope it's `onAssetHistoricalPrice` because this data is called on a case by case in the UI so it wouldn't benefit from batching

Reply:
Let me fix that PR then I can fix the SIP